Makefile

```diff
@@ -119,7 +119,6 @@ clean:
 deps:
 	go env
 	./etcd/install.sh $(TEST_ETCD_VERSION)
-	mkdir -p .bin
```

Comment:
Redundant as it is created by `./etcd/install.sh` (therefore clean target still contains `rm -rf .bin`)